ExtraHop/revealx_360/tests/test_3.json

```
@@ -0,0 +1,45 @@
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Reveal(x) 360",
+        "dialect_uuid": "1df44c62-33d3-41d4-8176-f1fa13589eea"
+      }
+    }
+  },
+  "expected": {
+    "message": "{\"id\": 33333333333, \"start_time\": 1701379823296, \"update_time\": 1706720535987, \"mod_time\": 1706720577690, \"title\": \"Weak Cipher Suite\", \"description\": \"[db1\\\\.example\\\\.org](#/metrics/devices/bcaa64bcd3c5440ea94d1b73c75979ae.0ed41b93cf2f0000/overview?from=1701379823&interval_type=DT&until=1706720940) negotiated an SSL/TLS session with a cipher suite that includes a weak encryption algorithm such as CBC, 3DES, RC4, null, anonymous, or export. Remove this cipher suite from [db1\\\\.example\\\\.org](#/metrics/devices/bcaa64bcd3c5440ea94d1b73c75979ae.0ed41b93cf2f0000/overview?from=1701379823&interval_type=DT&until=1706720940) and replace with stronger cipher suites.\", \"risk_score\": 30, \"type\": \"weak_cipher_individual\", \"recommended_factors\": [], \"recommended\": false, \"categories\": [\"sec\", \"sec.hardening\"], \"properties\": {\"cipher_suite\": \"TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA\"}, \"participants\": [{\"role\": \"offender\", \"object_id\": 44444444444, \"object_type\": \"device\", \"object_value\": \"1.2.3.4\", \"hostname\": \"db1.example.org\", \"id\": 2657, \"external\": false, \"scanner_service\": null}], \"ticket_id\": null, \"assignee\": null, \"status\": null, \"resolution\": null, \"mitre_tactics\": [], \"mitre_techniques\": [], \"appliance_id\": 3, \"is_user_created\": false}",
+    "event": {
+      "category": ["intrusion_detection"],
+      "code": "weak_cipher_individual",
+      "kind": "alert",
+      "reason": "[db1\\.example\\.org](#/metrics/devices/bcaa64bcd3c5440ea94d1b73c75979ae.0ed41b93cf2f0000/overview?from=1701379823&interval_type=DT&until=1706720940) negotiated an SSL/TLS session with a cipher suite that includes a weak encryption algorithm such as CBC, 3DES, RC4, null, anonymous, or export. Remove this cipher suite from [db1\\.example\\.org](#/metrics/devices/bcaa64bcd3c5440ea94d1b73c75979ae.0ed41b93cf2f0000/overview?from=1701379823&interval_type=DT&until=1706720940) and replace with stronger cipher suites.",
+      "start": "2023-11-30T21:30:23.296000Z",
+      "type": ["info"]
+    },
+    "@timestamp": "2024-01-31T17:02:57.690000Z",
+    "extrahop": {
+      "revealx360": {
+        "categories": ["sec", "sec.hardening"],
+        "id": "33333333333",
+        "title": "Weak Cipher Suite"
+      }
+    },
+    "host": {
+      "id": "44444444444",
+      "ip": "1.2.3.4",
```

This field must be an array (cf [specification](https://www.elastic.co/guide/en/ecs/current/ecs-host.html#field-host-ip))
```suggestion
      "ip": ["1.2.3.4"],
```